docs/user_guide/logging.rst

```diff
@@ -0,0 +1,57 @@
+Logging
+=======
+
+Since version 0.18.0 :program:`rebase-helper` uses standard module-level loggers.
+:program:`rebase-helper` uses :class:`rebasehelper.logger.CustomLogger` logger class which provides extra
+logging levels.
+
+Default console handlers can be setup by calling :meth:`rebasehelper.logger.create_stream_handlers` and default file
+handlers by calling :meth:`rebasehelper.logger.create_file_handlers`.
```

Comment:
I think it would be better to start this paragraph with something like `rebasehelper.logger module provides two utility functions to setup default handlers`.
Also, some description of what those default handlers actually are would be nice.